app.R

```diff
@@ -64,7 +70,13 @@ ui <- navbarPage("Sample Size Estimator",
                                 value = 0.8)
                 ),
                 mainPanel(
-                    plotlyOutput("plot_means")
+                    plotlyOutput("plot_means"),
+                    sliderInput("effect_size_range_mean",
+                                "Effect Size Range",
+                                width = "1000px",
```

Comment:
`width = "1000px"`, may have weird side effects if the window is small.  not sure if  `width = 100%` would help to solve the problem.

Reply:
Oh yeah good point! I'll try `width = 100%` and see if that works better

Reply:
Ok I added a commit to make that change